api/src/main/java/org/pmiops/workbench/institution/InstitutionEmailAddressMapper.java

```diff
@@ -0,0 +1,35 @@
+package org.pmiops.workbench.institution;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+import java.util.stream.Collectors;
+import org.mapstruct.Mapper;
+import org.pmiops.workbench.db.model.DbInstitution;
+import org.pmiops.workbench.db.model.DbInstitutionEmailAddress;
+import org.pmiops.workbench.model.Institution;
+import org.pmiops.workbench.utils.mappers.MapStructConfig;
+
+@Mapper(config = MapStructConfig.class)
+public interface InstitutionEmailAddressMapper {
+  default Set<DbInstitutionEmailAddress> modelToDb(
+      final Institution modelInstitution, final DbInstitution dbInstitution) {
+    return Optional.ofNullable(modelInstitution.getEmailAddresses()).orElse(Collections.emptyList())
```

Comment:
I'd try to guard against the list being nullable.

We noticed in [#workbench-techdesign](https://pmi-engteam.slack.com/archives/GP4CG4TFF/p1590099355134900) that some of our generated models are getting their lists initialized to new ArrayLists when they're declared, but others (sadly including yours) aren't. It would be great to get away from the need to do this check everywhere for a null collection.

Reply:
Do you have examples for the better models ?

Reply:
This showed up on a Friday afternoon and the best we could say about it was that it was "odd". I can take a look.

I still need to get back on OpenApi3 now that we're launched.